_layouts/default.html

```diff
@@ -65,15 +86,45 @@
 			padding: 3px;
 		}
 
+		/* Insets */
 		blockquote, pre {
 			margin-left: 3em;
 			overflow: auto;
 		}
 
-		@media only screen and (max-width: 800px) {
-			#wrapper {
-				margin: 1em;
-				width: auto;
+		blockquote > p {
+			margin: 0;
+		}
+
+		blockquote > p:not(:last-child) {
+			margin-bottom: 1em;
+		}
+
+		/* Smartphone-size viewports */
+		@media (max-width: 600px) {
+			ol, ul {
+				padding-left: 1em;
+			}
+			blockquote, pre {
+				margin-left: 1.5em;
+			}
+		}
+
+		/* Dark mode */
+		@media (prefers-color-scheme: dark) {
```

Comment:
I think it'd be better to define CSS variables in `:root` and override those variables in `@media (prefers-color-scheme: dark)`?

I've seen some projects maintain themes like this before, and it tends to lead to inconsistent use of colors, or broken themes on updates. ^-^'

Maintaining color pallets is a lot easier than overriding elements individually.

For example:

```css
:root {
  --pri-bg-color: #fff;
  --link-color: #0969da;
}

@media (prefers-color-scheme: dark) {
  :root {
    --pri-bg-color: #161b22;
    --link-color: #2f81f7;
  }
}

body {
  background-color: var(--pri-bg-color);
}

a:link, a:visited {
  color: var(--link-color);
}
```

Reply:
Using variables for values that're only referenced once is wasteful, IMHO. Especially for such a decidedly simple design.

It's your call, though.